synchronizer/block_range.go

```diff
@@ -0,0 +1,60 @@
+package synchronizer
+
+import (
+	"errors"
+	"fmt"
+)
+
+const (
+	latestBlockNumber  uint64 = ^uint64(0)
```

Comment:
```suggestion
	latestBlockNumber  uint64 = uint64(0)
```

Reply:
In fact is not an error is the negation of 0 that is 0xff......ffff